pkgs/development/python-modules/flask-seasurf/default.nix

```diff
@@ -12,6 +12,29 @@ buildPythonPackage rec {
     hash = "sha256-L/ZUEqqHmsyXG5eShcITII36ttwQlZN5GBngo+GcCdw=";
   };
 
+  patches = [
+    # Remove usage of deprecated flask._app_ctx_stack
+    (fetchpatch {
+      url = "https://github.com/maxcountryman/flask-seasurf/commit/9039764a4e44aeb1acb6ae7747deb438bee0826b.patch";
+      hash = "sha256-bVYzJN6MXzH3fNMknd2bh+04JlPJRkU0cLcWv+Rigqc=";
+    })
+  ];
+
+  postPatch = ''
+    # - cookie_jar is private in werkzeug 2.3, so recreate the client instead
+    # - set_cookie does not take a hostname argument anymore, use domain instead
+    substituteInPlace test_seasurf.py \
+      --replace '    client.cookie_jar.clear()' 'with self.app.test_client() as client:' \
+      --replace "client.set_cookie('www.example.com', self.csrf._csrf_name, token)" "client.set_cookie(self.csrf._csrf_name, token, domain='www.example.com')" \
+      --replace "c.delete('/foo/baz'," "c.delete('/foo/baz', content_type='application/json'," \
+
+    # Headers need to specify a content type
+    sed -i "s#\(post\|delete\)(\(u\?'[^']*'\(, headers=headers\)\?\))#\1(\2, content_type='application/json')#g" test_seasurf.py
+
+    # Disable some tests
+    sed -i "s#\(\(test_header_set_on_post\|test_https_good_referer\|test_https_referer_check_disabled\)(self):\)#\1\n        return#g" test_seasurf.py
```

Comment:
Do they really not support pytest?

Reply:
The fix is commonly to add the disabled tests to `disabledTests`

Reply:
Sorry, didn’t have time in the last weeks to look at this again (not yet sure if I will find time in the coming weeks).

I tried `disabledTests` and it didn’t work (I don’t remember which test system this uses but I didn’t even find a command line flag to skip tests).